### ISSUE TITLE
UI: Adding LDAP documentation report and Removing LDAP Exception

### ISSUE THREAD

**Issue by MejdiB** [apache/cloudstack#5610]
##### ISSUE TYPE
 * Bug Report
 * Documentation Report

##### COMPONENT NAME
~~~
UI, API
~~~

##### CLOUDSTACK VERSION
~~~
4.15.2
~~~

##### CONFIGURATION
A Cloudstack 4.15.2 environemnt is used and a separately installed LDAP server.

##### SUMMARY
- On the Cloudstack documentation, it seems like the only way to add an LDAP configuration can be done with cloudmonkey, what about using the UI?
- On the UI, after adding a basic LDAP configuration, I expected the "Add LDAP Account" button to manually add accounts to Cloudstack, but it does not happen..
- Removing an LDAP configuration via the GUI causes an exception 

##### STEPS TO REPRODUCE
Reproducing the exception:

Add an LDAP configuration on the UI under Configuration -> LDAP configuration and removing it will cause the error "com.cloud.exception.InvalidParameterValueException: Cannot find configuration with hostname [...]", although the information is stored in the db

##### EXPECTED RESULTS
~~~
- Appearance of Add LDAP Account button
- Removing LDAP configuration possible
~~~

**Comment by DaanHoogland** [comment]
@MejdiB can you test this on latest, and update your ticket description if something is still missing, please?

**Comment by DaanHoogland** [comment]
@MejdiB closing this as
Fixed by #5871
please check if all your concerns are met.
